scripts/truncate_fhd_files.pro

```diff
@@ -1,61 +1,114 @@
-pro truncate_fhd_files
```

Comment:
I don't speak IDL, so I'm just assuming this is all okay :)

Reply:
I tweaked this until I got the test files I wanted. I included the changes for posterity :)